styleguide/lib/searchfields.md

```diff
@@ -3,7 +3,11 @@ name: Search fields
 category: Form-Components
 ---
 
-A search field should be used to give a user the ability to search for options.
+Search box allows the user to locate content quickly and efficiently and can be seen as a conversation between the user and the web page.
+
+### ✅ When to use: 
+Smaller search fields are used on individual pages if they are to be searched within a specific area. It is important to set the user context for the search with the correct placeholder text.

```

Comment:
Hvorfor smaller text fields? Får de ikke være store?

Reply:
De kan säkert vara stora, men det bör vara en visuell balans mellan elementen i feks tabeller och text fields

Reply:
Det er alltid gitt, så kan eventuelt bare fjerne "Smaller search fields are used on individual pages if they are to be searched within a specific area." og fokusere mer på den neste settningen

Reply:
Vi vill ju helst att de skal bruke de större input fieldsen i den mån det går, men det kanske håller med att ta bort texten om det är självklart?

Reply:
"It is important to set the user context for the search with the correct placeholder text."